### DISCUSSION TITLE
PowerShell Community Call - October 19th 2023

### DISCUSSION THREAD

**Discussion by StevenBucher98** [PowerShell/PowerShell#20327] [Community Call]
Agenda
- PSResourceGet GA - @SydneyhSmith @anamnavi @alerickson 
- Stable Release of VSCode @andyleejordan 
- PSReadLine GA @StevenBucher98 
- 7.4 RC @SteveL-MSFT 
- OpenSSH 9.4 @tgauth 
- Az CLI/PS Updates @dcaro 
- Docs Update - @sdwheeler 
- GitHub Issues Updates Nov 1st @SteveL-MSFT 
- Ignite @StevenBucher98  @SydneyhSmith 
- November Community Call date change @StevenBucher98 @SydneyhSmith 
- PSSummit CFP - James Petty
- Community Demo/Announcements/Discussions

**Comment by floh96** [comment]
@StevenBucher98 the video of the call is private https://www.youtube.com/watch?v=xpEjE856P8M

**Reply by StevenBucher98** [reply]
Fixed!
